js/gimmicks/chart.js

```diff
@@ -0,0 +1,111 @@
+(function($) {
+    'use strict';
+
+    var scripturl = 'http://www.chartjs.org/assets/Chart.js';
+
+    var chartGimmick = {
+        name: 'chart',
+        version: $.md.version,
+        once: function() {
+            $.md.linkGimmick(this, 'chart', chart);
+
+            // load the chart js
+            $.md.registerScript(this, scripturl, {
+                license: 'EXCEPTION',
+                loadstage: 'skel_ready',
+                finishstage: 'bootstrap'
```

Comment:
Change the finishstage from 'boostrap' to 'gimmick' - That way the replacement of the links has finished at the 'gimmick' stage - (we will use that later)